src/index.js

```diff
@@ -0,0 +1,8 @@
+import getCityWeather from './weather_api';
+import createForm from './dom';
+
+const appId = 'f2526aabeeb777ad3fbb777842b7c51e';
+
+createForm(getCityWeather, appId);
+
+// console.log(getCityWeather('london', 'metric'));
```

Comment:
- [ ] Lets keep the code base clean and organized, remove the commented out code.